webapp/src/translationTools/useErrorTranslation.ts

```diff
@@ -151,6 +151,18 @@ export function useErrorTranslation() {
         return t('user_is_managed_by_organization');
       case 'cannot_set_sso_provider_missing_fields':
         return t('cannot_set_sso_provider_missing_fields');
+      case 'namespace_cannot_be_used_when_feature_is_disabled':
+        return t('namespace_cannot_be_used_when_feature_is_disabled');
+      case 'namespaces_cannot_be_disabled_when_namespace_exists':
+        return t('namespaces_cannot_be_disabled_when_namespace_exists');
+      case 'import_file_warning_header_namespace_cannot_be_used_when_feature_is_disabled':
+        return t(
+          'import_file_warning_header_namespace_cannot_be_used_when_feature_is_disabled'
+        );
+      case 'import_file_warning_message_namespace_cannot_be_used_when_feature_is_disabled':
+        return t(
+          'import_file_warning_message_namespace_cannot_be_used_when_feature_is_disabled'
+        );
```

Comment:
If we need specific messages for the import page (do we?). I'd move the translations to a separate hook.